src/Query.ts

```diff
@@ -21,8 +21,11 @@ export type FetchPolicy =
 export interface QueryOptions {
   raw?: boolean // If set, the response data is returned verbatim, rather than parsing them into the relevant MST models
   fetchPolicy?: FetchPolicy
+  noSsr?: boolean
 }
 
+const isServer: boolean = typeof window === "undefined"
```

Comment:
Might this come back as true for `react-native`?

Reply:
I'm not sure, but there's a good chance yes. But why would you use `ssr: true` for a react-native app? I believe with a react-native app you would have no place to use (and no use for) SSR.

Out of curiosity I spent a bit of time researching the correct way to define this, but couldn't find out anything certain of whether it would work with react-native. But I think it should not matter here.

Reply:
true! I made this comment before reading the below, so I think we're good :)

Reply:
Another option is `const isServer = !process.browser` (and is the way I like to do it, and I even use it inline) but that depends on using a module bundler that [shims the global `process` object](https://www.npmjs.com/package/process) (webpack & browserify do it by default) and also I got a [type error when trying to use it in typescript](https://github.com/zeit/next.js/issues/2177)